.pyup.yml

```diff
@@ -0,0 +1,63 @@
+# configure updates globally
+# default: all
+# allowed: all, insecure, False
+update: all
+
+# configure dependency pinning globally
+# default: True
+# allowed: True, False
+pin: True
+
+# set the default branch
+# default: empty, the default branch on GitHub
+branch: dev
+
+# update schedule
+# default: empty
+# allowed: "every day", "every week", ..
+schedule: "every day"
+
+# search for requirement files
+# default: True
+# allowed: True, False
+search: True
+
+# Specify requirement files by hand, default is empty
+# default: empty
+# allowed: list
+requirements:
+  - requirements/staging.txt:
+      # update all dependencies and pin them
+      update: all
+      pin: True
+  - requirements/dev.txt:
+      # don't update dependencies, use global 'pin' default
+      update: False
+  - requirements/prod.txt:
+      # update insecure only, pin all
+      update: insecure
+      pin: True
```

Comment:
We have no structure as described in this section (lines 28-39) but we do have our `requirements.txt` in a custom location (that is **not** `requirements/staging.txt`, `requirements/dev.txt` neither `requirements/prod.txt`). However as it's still called `requirements.txt` we might not need to manually list it in the pyuo.io config file.

You have to check their docs to make sure but as they've already opened a tons of PRs (linked in #299) I assume it's working as it is.